changelog/1548.feature.rst

```diff
@@ -0,0 +1,3 @@
+Updated the return value of the ``mass`` method of `~plasmapy.particles.particle_class.Particle`
+to return np.nan, rather than raising a MissingParticleDataError, when the mass is unavailable
+(e.g. for neutrinos or elements with no standard atomic weight).
```

Comment:
```suggestion
Updated the return value of the `~plasmapy.particles.particle_class.Particle.mass`
attribute of |Particle| to return |nan| kg, rather than a |MissingParticleDataError|, when 
the mass is unavailable (e.g. for neutrinos or elements with no standard atomic weight).
```
This should make it link to the docstring for `Particle.mass`.  I also applied some reST substitutions that we've defined in `docs/common_links.rst`.